tests/topotests/bgp_gr_functionality_topo3/bgp_gr_functionality_topo3.json

```diff
@@ -0,0 +1,222 @@
+{
+    "address_types": [
+        "ipv4",
+        "ipv6"
+    ],
+    "ipv4base": "20.20.20.0",
```

Comment:
Please use https://en.wikipedia.org/wiki/Reserved_IP_addresses (Documentation) ranges for IPv4 and IPv6.

Reply:
Hi @ton31337 , We have updated the IPs accordingly.
Thank you!